srcpkgs/python3-pygame/template

```diff
@@ -1,24 +1,19 @@
 # Template file for 'python3-pygame'
 pkgname=python3-pygame
-version=1.9.6
-revision=4
+version=2.0.1
+revision=1
 wrksrc="pygame-${version}"
 build_style=python3-module
 make_build_args="cython"
 hostmakedepends="pkg-config python3-setuptools python3-Cython
- SDL_mixer-devel SDL_image-devel SDL_ttf-devel libjpeg-turbo-devel portmidi-devel"
-makedepends="python3-devel SDL_mixer-devel SDL_image-devel
- SDL_ttf-devel libjpeg-turbo-devel portmidi-devel"
+ SDL2_mixer-devel SDL2_image-devel SDL2_ttf-devel libjpeg-turbo-devel portmidi-devel"
+makedepends="python3-devel SDL2_mixer-devel SDL2_image-devel
+ SDL2_ttf-devel libjpeg-turbo-devel portmidi-devel"
 short_desc="Collection of Python modules for writing games (Python3)"
 maintainer="Archaeme <normandy@firemail.cc>"
 license="LGPL-2.1-or-later"
 homepage="https://www.pygame.org/"
 distfiles="${PYPI_SITE}/p/pygame/pygame-${version}.tar.gz"
-checksum=301c6428c0880ecd4a9e3951b80e539c33863b6ff356a443db1758de4f297957
+checksum=8b1e7b63f47aafcdd8849933b206778747ef1802bd3d526aca45ed77141e4001
 
 export PORTMIDI_INC_PORTTIME=1
-
-pre_build() {
-	# Remove the C source to re-cythonize, which avoids some API changes
-	rm src_c/pypm.c src_c/_sdl2/{sdl2,audio,video}.c
-}
```

Comment:
Did they stop shipping these files or did re-generation stop being necessary?

Reply:
I believe the re-generation is no longer necessary.  I checked the ArchLinux package and they do not have this there and it appears to work without this.